src/components/markdown-render/inlineCode.js

```diff
@@ -10,7 +9,7 @@ const InlineCode = ({ children, className, additionalPreClasses, theme }) => {
       {...defaultProps}
       code={children}
       language={language}
-      theme={theme || customTheme}
+      theme={undefined}
```

Comment:
What happens if we make this undefined?

Reply:
If this is set to undefined, the `<Highlight>` component will not load a theme from `node_modules` and will instead fallback to just applying CSS classes to the code sample. And then our new CSS will kick in and apply the styles.

Reply:
I tried deleting the line and it didn't work. It needs to be explicitly set to `undefined` I guess.

Reply:
What benefits do we get from still using the Highlight component with this change?

Reply:
The Highlight component adds the Prism classnames that we use to theme the code. We still need it even if we're not using React theming.